returnn/datasets/basic.py

```diff
@@ -967,6 +967,26 @@ def num_seqs(self) -> int:
         """
         raise NotImplementedError
 
+    def get_epoch_continuous(self, sorted_seq_idx: int) -> float:
```

Comment:
The name here is wrong. `epoch_continuous` is what you have here added to `epoch - 1`.

However, it makes sense to calculate what you do here. So do not change the logic, just change the name of this function (and similarly also below for other uses of "epoch_continuous"). I suggested "complete_frac" before. Not sure. So maybe:
```suggestion
    def get_complete_frac(self, sorted_seq_idx: int) -> float:
```